shop.py

```diff
@@ -83,34 +91,58 @@
 ]
 
 class NewShop:
-  def __init__(self, atk, deff, heal):
+  def __init__(self, item1, item2, item3):
     
-    self.name1 = weaponList[atk]['itemName']
-    self.name2 = armorList[deff]['itemName']
-    self.name3 = healingList[heal]['itemName']
-    self.atk = weaponList[atk]['value']
-    self.deff = armorList[deff]['value']
-    self.heal = healingList[heal]['value']
+    a = weaponList[item1]
+    b = armorList[item2]
+    c = healingList[item3]
+
+    global item1name, item2name, item3name, item1value, item2value, item3value
+
+    item1name = a['itemName']
+    item2name = b['itemName']
+    item3name = c['itemName']
+
+    item1value = a['value']
+    item2value = b['value']
+    item3value = c['value']
+
+    print(f"Item 1 name is {item1name}")
+
+    print(f"printing list a {a}")
+    print(f"printing list b {b}")
+    print(f"printing list c {c}")
+
+    self.shop = {
+      0:a,
+      1:b,
+      2:c
+
+    }
 
   def showValue(self):
     return(
-      "=====================================\n"
-      f"Pick one of the items to go with your adventure\n"
-      f"1-Weapon : {self.name1} , increase attack by {self.atk} \n"
-      f"2-Armor : {self.name2} , increase defense by {self.deff} \n"
-      f"3-Healing : {self.name3} , recover HP by {self.heal} \n"
-      "=====================================\n"
-      
+      "================================\n"
+      "Pick one of the following items\n"
+      f"1- Weapon : {item1name} , increase attack by {item1value}\n"
```

Comment:
` f"1- Weapon : {item1name} , increase attack by {item1value}\n"` to ` f"1- Weapon : {self.shop[0]['name']} , increase attack by {self.shop[0]['value']}\n"`